src/backend/distributed/commands/call.c

```diff
@@ -193,8 +194,8 @@ CallFuncExprRemotely(CallStmt *callStmt, DistObjectCacheEntry *procedure,
 			ROW_MODIFY_NONE, list_make1(task), MaxAdaptiveExecutorPoolSize,
 			localExecutionSupported
 			);
-		executionParams->tupleStore = tupleStore;
-		executionParams->tupleDescriptor = tupleDesc;
+		executionParams->tupleDestination = CreateTupleStoreTupleDest(tupleStore,
+																	  tupleDesc);
```

Comment:
Below we seem to be emitting the tuple store into a(n already initialised) DestReceiver. Could we have a DestReceiverTupleDest? This might be useful in other cases as well.